src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -173,15 +176,26 @@ class HuiMapCard extends LitElement implements LovelaceCard {
             interactive-zones
             render-passive
           ></ha-map>
-          <ha-icon-button
-            .label=${this.hass!.localize(
-              "ui.panel.lovelace.cards.map.reset_focus"
-            )}
-            .path=${mdiImageFilterCenterFocus}
-            style=${isDarkMode ? "color:#ffffff" : "color:#000000"}
-            @click=${this._fitMap}
-            tabindex="0"
-          ></ha-icon-button>
+          <div id="buttons">
+            <ha-icon-button
+              .label=${this.hass!.localize(
+                "ui.panel.lovelace.cards.map.toggle_grouping"
+              )}
+              .path=${mdiGoogleCirclesCommunities}
```

Comment:
@jpbede what about having the icon `dots-hexagon` when clusterMarkes is disabled?
I don't like the fact that you cannot see in the UI what settings are currently active.

Reply:
Yea, true, will update the PR. Thanks for the suggestion 👍